src/SDKExecutor.js

```diff
@@ -65,7 +65,7 @@ module.exports.SDKExecutor = class SDKExecutor {
 				? SDK_INTEGRATION_MODE_JVM_OPTION
 				: '';
 
-			const developmentModeOption = SDK_DEVELOPMENT_MODE_JVM_OPTION; //HARDCODED: this should be in the SDK
+			const developmentModeOption = executionContext.isDevelopmentMode() ? SDK_DEVELOPMENT_MODE_JVM_OPTION : '';
```

Comment:
The final decision was that the Node.js CLI would not need to send this flag to the SDK since each authentication stored in the credentials file will have the "dev: true". Then, the SDK can know by itself (based on the current auth) wether it's running in dev mode or not.
Therefore, this `developmentModeOption` should be removed in the future. 

Instead, the `authenticate` command needs a new flag to create authentications in `dev` mode.